deploy/aws/base-install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
Can we add a `set -o errexit` to all our shell scripts so that the scripts don't continue executing when a command fails?

```suggestion

set -o errexit

```

Reply:
Addedd the "set -o errexit" in each script